pkg/router/template/router.go

```diff
@@ -812,6 +812,20 @@ func (r *templateRouter) RemoveRoute(route *routeapi.Route) {
 	r.stateChanged = true
 }
 
+// numberOfEndpoints returns the number of endpoints
+func (r *templateRouter) numberOfEndpoints(id string) int32 {
+	r.lock.Lock()
+	defer r.lock.Unlock()
+
+	var eps int32
```

Comment:
eps := 0

(0 is a type int, and that it as least 32 bits)

Or if you must:

var eps int32 = 0